README.md

```diff
@@ -48,7 +48,7 @@ These are the various QR Code generation options provided by [rqrqcode_core](htt
 ```
 string - the string you wish to encode
 
-size   - the size of the qrcode (default 4)
+size   - the size of the qrcode (defaults to smallest size needed to encode the string)
```

Comment:
I wonder if we can use `the size (Integer) of the qrcode (defaults to smallest size needed to encode the string)` so that the user continues to know the option is a number?

Reply:
```suggestion
size   - the size (Integer) of the qrcode (defaults to smallest size needed to encode the string)
```
Sure!